.gitmodules

```diff
@@ -12,7 +12,7 @@
 	branch = support/HAFS
 [submodule "gfdl-tracker"]
 	path = sorc/hafs_vortextracker.fd
-	url = https://github.com/hafs-community/gfdl-tracker.git
+        url = https://github.com/hafs-community/gfdl-tracker.git
```

Comment:
@BijuThomas-NOAA, please fix the indentation for this url line above. Thanks! Also make sure it points to the latest support/HAFS branch for sorc/hafs_vortextracker.fd.

Reply:
@BijuThomas-NOAA, please revert this empty space change, and also update the hafs_vortextracker.fd submodule by:
cd sorc/hafs_vortextracker.fd
git checkout support/HAFS
git pull
cd ../
git add  hafs_vortextracker.fd
git commit
then git push
After these, I think this PR is ready to be merged. Please feel free to let me know if you have any question. Thanks! Bin

Reply:
@BinLiu-NOAA,
I don't see the empty space locally when I open with vi
[submodule "gfdl-tracker"]
        path = sorc/hafs_vortextracker.fd
        url = https://github.com/hafs-community/gfdl-tracker.git
        branch = support/HAFS